### DISCUSSION TITLE
Tiles option to make tall terrain and furniture sprites semi-transparent above a certain point

### DISCUSSION THREAD

**Discussion by int-ua** [CleverRaven/Cataclysm-DDA#45925] [Ideas]
Some tall sprites in Ultica cover northern tile too much, making it easy to miss things behind them. But even one of the sprite authors opposed the idea to make trees semi-transparent and I can't deny him the ability to enjoy tileset art to the fullest by just forcing transparency on everyone. Would it be too hard to implement this as an option in the game?

### Additional context

https://github.com/I-am-Erk/CDDA-Tilesets/issues/327

https://discord.com/channels/552510581161066497/552510581782085654/785511928575295508

**Comment by zantanzuken** [comment]
gonna put it here cause i put it there, but:

the best implementation of this (in my opinion at least) would be to create a system in cataclysm that semi-transparent's tiles (mostly trees, sometimes skeletal juggernauts, etc.) when they have something behind them (i.e. a monster/animal/humanoid, not an item) 

... course i have no idea if/how anyone would do that.
